apps/labs/posts/pytorch-tensoriterator-internals.md

```diff
@@ -0,0 +1,276 @@
+---
+title: 'PyTorch TensorIterator Internals'
+published: April 13, 2020
+author: sameer-deshmukh
+description: 'This post is a deep dive into how TensorIterator works, and is an essential part of learning to contribute to the PyTorch codebase since iterations over tensors in the C++ codebase are extremely commonplace. This post is aimed at someone who wants to contribute to PyTorch'
+category: [Machine Learning]
+featuredImage:
+  src: posts/pytorch-tensoriterator-internals/feature.png
+  alt: 'Code snippet on creating a TensorIterator using a default constructor.'
+hero:
+  imageSrc: posts/pytorch-tensoriterator-internals/blog_hero_var2.svg
+  imageAlt: 'An illustration of a dark brown hand holding up a microphone, with some graphical elements highlighting the top of the microphone.'
+'
```

Comment:
```suggestion
```

Based on r[ecent Vercel build error](https://vercel.com/quansight/quansight-labs-live/2Z7sZW85bn2byHAAS2siN5nS4XCU), `error YAMLException: unexpected end of the stream within a single quoted scalar at line 14, column 1:`